### ISSUE TITLE
WordPress.org and plugin releases

### ISSUE THREAD

**Issue by slaFFik** [spatie/wordpress-ray#19]
I see that you made a release 1.0.1:
![image](https://user-images.githubusercontent.com/282234/104491149-82c34980-55da-11eb-8141-5313293a327a.png)

But WordPress.org is not aware of it and still allows everyone to download 1.0.0 by default:
![image](https://user-images.githubusercontent.com/282234/104491218-9d95be00-55da-11eb-8c80-7d992dba17e7.png)

That's because in your `trunk/readme.txt` you still point to the `1.0` which automatically instructs WP.org to zip and distribute the `tags/1.0.0`.

To make the 1.0.1 release go live and instruct WP.org to use `tags/1.0.1` as the source of the zip you need to update in `trunk/readme.txt` the `Stable tag: 1.0` to `Stable tag: 1.0.1`. Just creating a new directory inside SVN `tags/` is not enough
And you will need to do this step for each and every release.

Basically, your `trunk/*` can always be the same as the latest tag you want to release. And for each release, you need to manually increment the `Stable tag: X.X.X` value inside the `readme.txt`.
You can leave in `trunk/` only one `readme.txt` file and remove all the rest (and still update it and increment the `Stable tag` for each release), but I find it easier to just copy the release-prepared code to both `tags/X.X.X` and `trunk/`.

Here is the interface I used to identify this issue: https://plugins.trac.wordpress.org/browser/spatie-ray/

**Comment by freekmurze** [comment]
Thanks, I hope I've done everything right in 1.0.2 🙂
